src/connectors/wire/WireProtocol.cpp

```diff
@@ -231,7 +232,7 @@ namespace {
             jsonOutput.clear();
             response.accept(*this);
             const mValue v(jsonOutput);
-            return write_string(v, false);
```

Comment:
Shouldn't this commit also change at least `WireProtocolTest` to make it pass again?